src/2017/QT_Build.org

```diff
@@ -0,0 +1,73 @@
+#+TITLE: Using QT/Build Systems
+#+AUTHOR: Jay Kamat, Matthew Woodward
+#+EMAIL: jaygkamat@gmail.com
+#+REVEAL_THEME: black
+#+REVEAL_TRANS: linear
+#+REVEAL_SPEED: fast
+#+REVEAL_PLUGINS: (notes pdf)
+#+REVEAL_HLEVEL: 1
+#+OPTIONS: toc:nil timestamp:nil reveal_control:t num:nil reveal_history:t tags:nil author:nil
+
+# Export section for md
+* QT Introduction {#t20176}                                            :docs:
```

Comment:
this hash should be changed to a uid of the file, for linking purposes in the docs. It just needs to be unique.